core/controllers/editor.py

```diff
@@ -785,6 +789,16 @@ def post(self, unused_exploration_id):
         user_services.record_user_started_state_editor_tutorial(self.user_id)
 
 
+class StartedTranslationTutorialEventHandler(EditorHandler):
```

Comment:
Can you please add a test for this handler?

Reply:
where do you handle tests for the editor?